src/components/Button/Button.js

```diff
@@ -0,0 +1,28 @@
+export default class Button {
+  constructor(text, additionalClassName, clickEvent) {
+    this._text = text;
+    this._additionalClassName = additionalClassName;
+    this._clickEvent = clickEvent;
+
+    this.$button = null;
+  }
+
+  render() {
+    if (!this.$button) {
+      this.$button = document.createElement("button");
+      this.$button.setAttribute("type", "button");
+      this.$button.classList.add("button");
+      if (this._additionalClassName) {
+        this.$button.classList.add(this._additionalClassName);
+      }
+
+      this.$button.addEventListener("click", (e) => {
+        this._clickEvent();
+      });
```

Comment:
이벤트 객체 사용 안한다면, 함수 바로 넣어줘도 될것 같아요
```suggestion
      this.$button.addEventListener("click", this._clickEvent);
```

Reply:
앗 지적 감사합니다 ㅠㅠ 수정하다가 완벽히 정리가 안 된 부분이 많군요..